R/run_stress_test.R

```diff
@@ -22,17 +22,20 @@
 #'   range compare `term_range_lookup`.
 #' @param company_exclusion Boolean, indicating if companies provided in dataset
 #'   excluded_companies.csv shall be excluded.
+#' @param asset_type String holding asset_type, for allowed value compare
+#'   `asset_types_lookup`.
 #' @return NULL
 #' @export
-run_stress_test_loans <- function(lgd_senior_claims = 0.45,
-                                  lgd_subordinated_claims = 0.75,
-                                  terminal_value = 0,
-                                  risk_free_rate = 0.02,
-                                  discount_rate = 0.02,
-                                  div_netprofit_prop_coef = 1,
-                                  shock_year = 2030,
-                                  term = 2,
-                                  company_exclusion = TRUE) {
+run_stress_test <- function(lgd_senior_claims = 0.45,
+                            lgd_subordinated_claims = 0.75,
+                            terminal_value = 0,
+                            risk_free_rate = 0.02,
+                            discount_rate = 0.02,
+                            div_netprofit_prop_coef = 1,
+                            shock_year = 2030,
+                            term = 2,
+                            company_exclusion = TRUE,
+                            asset_type) {
```

Comment:
A common pattern is to first give arguments that have no default, then the "details" arguments that have defaults.

https://design.tidyverse.org/call-data-details.html

Reply:
DONE